southbound/impl/src/main/java/org/opendaylight/ansible/southbound/AnsibleCommandServiceImpl.java

```diff
@@ -56,7 +68,14 @@ public AnsibleCommandServiceImpl(@OsgiService final DataBroker dataBroker) {
         Status status;
         String failedEvent;
         try {
-            uuid = runAnsible(input.getHost(), input.getDirectory(), input.getFile());
+            if (input.getCommandType() instanceof Playbook) {
+                uuid = runAnsiblePlaybook(input.getHost(), input.getDirectory(), ((Playbook) input.getCommandType()).getFile(),
```

Comment:
line is more than 120 so it fails checkstyle

Reply:
probably need to figure out why my setup fails the northbound impl test cases so that I can actually hit these checkstyle issues.

Reply:
ah ok this works and I am able to hit he issues